unittest {
    try
        doStuff();
    catch (Exception ex)
        complain(ex.msg);
}
